lib/redis_memo/memoize_query/cached_select/bind_params.rb

```diff
@@ -2,47 +2,107 @@
 
 class RedisMemo::MemoizeQuery::CachedSelect
   class BindParams
-    def params
-      #
-      # Bind params is hash of sets: each key is a model class, each value is a
-      # set of hashes for memoized column conditions. Example:
-      #
-      #   {
-      #      Site => [
-      #        {name: 'a', city: 'b'},
-      #        {name: 'a', city: 'c'},
-      #        {name: 'b', city: 'b'},
-      #        {name: 'b', city: 'c'},
-      #      ],
-      #   }
-      #
-      @params ||= Hash.new do |models, model|
-        models[model] = []
-      end
+    def initialize(left = nil, right = nil, opt = nil)
+      @left = left
+      @right = right
+      @opt = opt
+      @plan = nil
```

Comment:
can we comment and explain here that the plan is used to generate a plan and determine whether to cache the query?